odra-test/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+#![no_std]
+#![cfg(not(target_arch = "wasm32"))]
+
+use odra_casper_test_vm::{CasperHost, CasperVm};
+use odra_core::prelude::String;
+use odra_core::HostEnv;
+use odra_vm::{OdraVm, OdraVmHost};
+
+pub fn test_env() -> odra_core::HostEnv {
```

Comment:
```suggestion
pub fn env() -> odra_core::HostEnv {
```
this way, when it use it looks like `odra_test::env()`